src/bundles/index.js

```diff
@@ -25,7 +25,7 @@ export default composeBundles(
   peerLocationsBundle(),
   routesBundle,
   redirectsBundle,
-  filesBundle,
+  filesBundle({ navbar: '#navbar' }),
```

Comment:
I'd like to find a css solution to this. Coupling the files bundle to the dom structure is a code smell.

Reply:
Yes, but for now I can't see any other way. If you have an idea, ping me.

Reply:
When I'm faced with a problem like this, I create a minimal dom and css example on jsbin and try to figure out what is the simplest thing that could work with just css for layout, and js for the actions.

Failing that, redux is the single source of truth for the application state. If we have to do layout in js and we have to know the width of the navbar, then it should be pulled up into the redux store and updated and pulled from there. We could keep it just in React if you think that's prefeable, but it should till be capured as state and pulled up high enough in the tree, that it can be passed into the components that need it. 

Just keep repeating "data flows one way" and "UI as a function of state" until you either go mad or reach a zen like enlightenment 🎐

Reply:
@hacdias can we remove the ({ navbar: '#navbar'}) param from the filesBundle function call now?

Reply:
Done!